src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js

```diff
@@ -130,6 +182,18 @@
         });
 
         updateUI();
+
+        $(function () {
```

Comment:
This should really be a directive. Any native DOM stuff should happen inside a directive to bridge between the angular world and the DOM world.

If we refactor some of the html markup, it's not obvious that this controller needs any of the `endpoint-data-changed` etc. items in the markup. If this was a directive like `<sticky-loading-indicator></sticky-loading-indicator>` or something, it becomes much more obvious what is happening in the DOM world.